lib/rspec/core/formatters/bisect_progress_formatter.rb

```diff
@@ -26,17 +28,40 @@ def bisect_original_run_complete(notification)
           output.puts "Starting bisect with #{failures} and #{non_failures}."
         end
 
+        def bisect_dependency_check_started(_notification)
+          output.print "Checking that failures are order-dependent.."
+        end
+
+        def bisect_dependency_check_finished(notification)
+          is_part = notification.dependent_failures ? "is" : "is not"
+          output.puts " failure #{is_part} order-dependent"
+        end
+
         def bisect_round_started(notification, include_trailing_space=true)
-          search_desc = Helpers.pluralize(
-            notification.subset_size, "non-failing example"
-          )
+          range_desc = [
+            notification.candidates_range.begin,
+            notification.candidates_range.end,
+          ].join("-")
 
-          output.print "\nRound #{notification.round}: searching for #{search_desc}" \
-                       " (of #{notification.remaining_count}) to ignore:"
+          output.print "\nBisecting over non-failing examples #{range_desc}"
           output.print " " if include_trailing_space
         end
 
-        def bisect_round_finished(notification)
+        def bisect_ignoring_ids(notification)
+          ignore_desc =
+            if notification.ignore_range.one?
+              "example #{notification.ignore_range.begin}"
+            else
+              "examples #{notification.ignore_range.begin}-" \
+                "#{notification.ignore_range.end}"
+            end
```

Comment:
I'm seeing duplication between the `range_desc = [ ... ].join('-')` bit in the method above and this bit here.  WDYT about passing a custom type instead of a `Range` object that has a `description` method that takes care of this formatting?  After all, we don't actually need any of the range semantics....just an object with a begin and an end.